integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/withapim/InterceptorWithMtlsTestCase.java

```diff
@@ -0,0 +1,148 @@
+package org.wso2.choreo.connect.tests.testcases.withapim;
```

Comment:
Missing license

Reply:
Thanks for noticing. fixed with https://github.com/wso2/product-microgateway/pull/2611/commits/6ff8e7d55d85dc362095542cce2f14efbdcfd023